rentals.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date
```

Comment:
you have to make book and person accessible e.g `attr_accessor :date, :book, :person`